test/integration/targets/canonical-pep517-self-packaging/minimum-build-constraints.txt

```diff
@@ -0,0 +1,19 @@
+# Lowest set in `pyproject.toml`:
+setuptools == 39.2.0; python_version < "3.9"
+
+# Lowest supporting Python 3.10:
+setuptools == 57.0.0; python_version == "3.9" or python_version == "3.10"
+
+# Lowest supporting Python 3.11:
+setuptools == 60.0.0; python_version >= "3.11"
+
+
+# The last version at the time of Python 3.9 release:
```

Comment:
Can you clarify this comment? I thought you might have meant "The latest version", but that doesn't appear to be the case.

Reply:
Yes, it looks like I was looking at the wrong year, I'll change it to 0.35.1, then.

Reply:
Turned out that py39 explodes with this version, maybe that's why I didn't go with it originally. Looks like I'm going to have to put in conditional pins for this as well...

Reply:
Ended up with 0.33.6 that is old enough and works for the whole range.